examples/cloud-router-2-aws-connection/README.md

```diff
@@ -233,7 +232,7 @@ provider "aws" {
   region     = var.zside_seller_region
 }
 module "cloud_router_aws_connection" {
-  source = "equinix/fabric/equinix//modules/cloud-router-connection"
+  source = "../../modules/cloud-router-connection"
```

Comment:
Ah, one last issue. This shouldn't change in the example files. You can use a sed command for this `sed -i .bak "s@\.\.\/\.\.@equinix\/fabric\/equinix\/@" $d/new_content.txt`

Might need to be altered for your specific GHA job and whatever you are using locally.

Reply:
Addressed with [this](https://github.com/equinix/terraform-equinix-fabric/pull/102/commits/e951b3db459c35d44702bf5141459852271c459a) commit